### DISCUSSION TITLE
error message snap is not installed

### DISCUSSION THREAD

**Discussion by saicharan0124** [MetaMask/snaps#1195] [Q&A]
so im running the snap in port 3000 and in my frontend i called by replacing the snap id with the link but i am getting the error snap is not installed when i invoke the snap what might be causing the problem am i calling the snap in a correct manner ? the snap file is actually a js template repo and i changed it a bit

**Comment by Montoya** [comment]
Hi, you need to be running both the snap and the website. You can either run them in separate terminal instances or if you use our typescript template it will run both packages from the main directory with just `yarn start`.

**Comment by saicharan0124** [comment]
I am running both the things still i m having the error
